GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -47,9 +66,9 @@ public void CanParseDiffForwards()
             DiffHelper diffForwards = new DiffHelper(tracer, new MockGVFSEnlistment(), new List<string>(), new List<string>());
             diffForwards.ParseDiffFile(GetDataPath("forward.txt"), "xx:\\fakeRepo");
 
-            // File added, file edited, file renamed, folder => file, edit-rename file
+            // File added, file edited, file renamed, folder => file, edit-rename file, SymLink added
             // Children of: Add folder, Renamed folder, edited folder, file => folder
-            diffForwards.RequiredBlobs.Count.ShouldEqual(9);
+            diffForwards.RequiredBlobs.Count.ShouldEqual(10);

```

Comment:
Related to above comment on test coverage, will `diffForwards.FileAddOperations` be different based on `SymLinkSupport`?